Data/member.json

```diff
@@ -414,5 +414,12 @@
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
         }
+        {
+            "name": "Abhyuday Singh",
+            "avatar": "https://avatars.githubusercontent.com/u/70817539?v=4",
+            "description": "Web Developer | ML Enthausiast | Anime Fan",
+            "githuburl": "https://github.com/abhyuday26",
+            "linkedinurl": "https://www.linkedin.com/in/abhyuday-singh-9778981a5/"
+        },
```

Comment:
@abhyuday26 As it is the end of the file, please can you remove the comma after the bracket? thanks